src/toil/cwl/cwltoil.py

```diff
@@ -729,10 +738,11 @@ def __init__(self, tool, cwljob, runtime_context, step_inputs=None, conditional=
                 force_docker_pull=False,
                 loadListing=u'',
                 outdir=u'',
-                tmpdir=u'',
-                stagedir=u''
+                tmpdir=u'/tmp',  # TODO: use actual defaults here
+                stagedir=u'/var/lib/cwl'  # TODO: use actual defaults here
```

Comment:
We don't need the `u` prefix anymore I don't think.

Reply:
Aye, it's redundant.  I'll clear those out.